tools/download_model_artifacts.py

```
@@ -0,0 +1,53 @@
+
+    options.add(file_object_name_parts[1])
+
+logger.info("Available artifacts", options=options)
+
+selected_option = input("Select an artifact to download: ")
+
+logger.info("Selected artifact", selected_option=selected_option)
+
+target_path = f"artifacts/{selected_option}/output/model.tar.gz"
+destination_directory = f"applications/{application_name}/src/{application_name}/"
+destination_path = os.path.join(destination_directory, "model.tar.gz")  # noqa: PTH118
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `PTH118`)

Remove unused `noqa` directive

(RUF100)